cli/src/commands/next.rs

```diff
@@ -127,11 +131,20 @@ pub(crate) fn cmd_next(
             // unnecessary now that --edit is implied if `@` has descendants.
             .minus(&wc_revset)
     };
+    let target_revset = if conflict {
+        target_revset
+            .descendants()
```

Comment:
If there are multiple descendants with conflicts, we should just choose the first ones, but it it looks like the code will return all of them.

Maybe it will be easier to read the code like this:

```rs
let start_revset = if edit {
  wc_revset
} else {
  wc_revset.parents()
};
let target_revset = if args.conflict {
  start_revset.descendants().conflicts().roots()
} else {
  start_revset.descendants_at(args.offset)
}.minus(&wc_revset);
```

Reply:
thanks, that's a nice simplification.

Reply:
Not done, it seems. Maybe you forgot to push your updates?

Reply:
ok, actually done. Previously I only took the revset expression, so that's my fault.

Reply:
Still not done in `prev.rs` :)